twamp/main.go

```diff
@@ -55,7 +57,7 @@ func main() {
 
 	switch *mode {
 	case "json":
-		results := test.RunX(*count)
+		results := test.RunX(*count, nil)
```

Comment:
I am not sure what needs to happen here so keeping the PR in draft mode while I figure it out.

Reply:
You can pass in an optional callback function here.  Passing in nil should be fine.